test/tools/README.md

```diff
@@ -28,7 +28,17 @@ export TEST_PROVISION_FILE=${PROPERTIES_FILE_PATH}
 export TEST_PROVISION="yes"
 ./caa-provisioner-cli -actions=${ACTION}
 ```
-`ACTION` supports `provision`, `deprovision` and `uploadimage`.
+`ACTION` supports `provision`, `deprovision`, `install`, `uninstall` and `uploadimage`.
+
+### Brief Action Explanations
+
+`provision` : Uses the provisioner to create a cluster
```

Comment:
Sorry, I thought I commented this in the last review, but can't spot it now. It's probably worth documenting that `provision` also does the `install` action, not just creates the cluster